Scripts/WaterPlayer.gd

```diff
@@ -0,0 +1,10 @@
+extends "res://Scripts/Player.gd"
+
+class_name WaterPath
+
+func _ready():
+	default_speed = 220
+
+func jump():
```

Comment:
Again, why are we overriding this method, it's identical to Player's implementation.  (to be fair I noticed I did this in FirePlayer too)